monorepo/monorepo-ws/apps/articleapp/src/configuration/config.ts

```diff
@@ -1 +1,3 @@
-export const port: number = Number(process.env.PORT)
\ No newline at end of file
+import { checkNumber } from "./utils";
+
+export const port: number = checkNumber(process.env.PORT)
```

Comment:
osobně bych sem dal default, ať se ty porty nemusej uvádět ve všech možnejch konfigách. 
```suggestion
export const port: number = checkNumber(process.env.PORT || 80)
```

Reply:
obecně využívat v konfigách nějaký společný defaulty, ať skutečný konfigurace nemusejí být dlouhý. 
V envech pak se můžou uvádět jen věci který jsou různý pro jednotlivý prostředí

Reply:
opraveno

Reply:
tyhle defaulty by se dali nastavit i pro cesty k souborům, itervaly a podobně.

Reply:
akorát jsem to myslel nejen pro port, ale i pro ostatní kde to dává smysl. třeba ty relativní cesty k soborům, nebo intervaly.

Reply:
Přidáno.